packages/paste-core/components/file-picker/src/FilePicker.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Text} from '@twilio-paste/text';
+import {useUID} from '@twilio-paste/uid-library';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {SiblingBox} from '@twilio-paste/sibling-box';
+
+export interface FilePickerProps extends React.HTMLAttributes<HTMLInputElement>, Pick<BoxProps, 'element'> {
+  accept?: string;
+  children: React.ReactElement;
+  disabled?: boolean;
+  i18nNoSelectionText?: string;
+  required?: boolean;
+}
+
+const FilePicker = React.forwardRef<HTMLInputElement, FilePickerProps>(
+  (
+    {
+      element = 'FILEPICKER',
```

Comment:
suggestion: should we make this FILE_PICKER, since it's two words?

Reply:
totally can, but I followed the date picker and time picker patterns of making it one word (prob should've split those in hindsight)

Reply:
ah gotcha gotcha